android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/AuthenticationManager.java

```diff
@@ -571,9 +574,6 @@ public ConsentSignature storeLocalConsent(@NonNull String subpopulationGuid, @No
                                               @Nullable String imageMimeType,
                                               @NonNull SharingScope sharingScope) {
         checkNotNull(subpopulationGuid);
-        checkNotNull(name);
```

Comment:
Removing these fixes AA-9, so we can save incomplete consent signatures

Reply:
Was AA-9 not fixed? It should've been fixed here: https://github.com/Sage-Bionetworks/BridgeAndroidSDK/pull/58/files

storeLocalConsent(String, ConsentSignature) is the one for saving incomplete consent signatures. This method is supposed to store completed ones. I'd argue part of the confusion is we should have two separate stores -- for completed and incompleted consents. Storing completed consents is a cache for uploading. Storing incompleted consents is for saving data from various steps towards building a completed consent.